### ISSUE TITLE
How can I aling the widget in the main screen?

### ISSUE THREAD

**Issue by manurafal** [mikesilvo164/Aeon-Nox-SiLVO#85]
How can I align the widget on the main screen? I need to put a widget in the center, but I do not know how.  And the same with the submenus, I need the submenus to be centered.

**Comment by jewbmx** [comment]
Skin settings / Main Menu / (for a few settings your probably wanting.) Then continue on to that top option called setup main menu and pick the item then the widget. Continue thru the process and your done. (Compact panel is pretty centered)

**Comment by manurafal** [comment]
I only have 2 widgets, and they appear in the left side. It is not short enough to center the 2 widgets.

**Comment by manurafal** [comment]
![imagen](https://user-images.githubusercontent.com/47554729/55091669-8e661900-50b1-11e9-9983-ff247702c2a1.png)

**Comment by manurafal** [comment]
![imagen](https://user-images.githubusercontent.com/47554729/55091829-cbcaa680-50b1-11e9-9ce4-1304c63764b5.png)

**Comment by manurafal** [comment]
And it happens the same with the submenus.

**Comment by jewbmx** [comment]
I started using a system of 3 because it works best for the skin. Since it is the only good skin for kodi i'll compromise lol. Kind of easy too when you see those 1, 3, 5 options

**Comment by manurafal** [comment]
I have not understood you well. Could you tell me how to center widgets and submenus ???

I am Spanish, and I do not know what it means "comact", Can you tell me what it is??

Thanks for all.

**Comment by manurafal** [comment]
How can I align the widget on the main screen? I need to put a widget in the center, but I do not know how. And the same with the submenus, I need the submenus to be centered.

**Comment by jewbmx** [comment]
Already told you lol, use 1, 3, or 5 items to get center and in some cases changes style type.

**Comment by Wanderer189** [comment]
This is not a problem, but request for help. I have posted in the forums like 3 times over past few months and never received a reply. I need to change the buffered portion color on the progress bar. There is a setting in the skin to change the played portion, but not the buffered portion. I have tried to find it in the 16x9 files with no success, Tried using the texture tool, but not sure what files it would be if it is using something in there, any help would be so much appreciated.
